agent/fluentconfigurationgenerator/generator/generator.go

```diff
@@ -0,0 +1,139 @@
+package generator
+
+import (
+	"io"
+	"text/template"
+)
+
+type IncludePosition int
```

Comment:
exported names should have comments (https://github.com/golang/go/wiki/CodeReviewComments#doc-comments)

Reply:
Oh yeah... I meant to do this but then forgot 👍🏻